lms/djangoapps/oauth2_handler/tests.py

```diff
@@ -158,43 +164,63 @@ def get_with_claim_value(self, scope, claim, values):
         self.assertEqual(result.status_code, 200)
         return claims
 
-    def test_request_staff_courses_using_scope(self):
-        CourseStaffRole(self.course_key).add_users(self.user)
-        claims = self.get_with_scope('course_staff')
-
-        courses = claims['staff_courses']
+    def _assert_role_using_scope(self, scope, claim, assert_one_course=True):
+        with check_mongo_calls(0):
+            claims = self.get_with_scope(scope)
+        courses = claims[claim]
         self.assertIn(self.course_id, courses)
-        self.assertEqual(len(courses), 1)
+        if assert_one_course:
+            self.assertEqual(len(courses), 1)
 
-    def test_request_instructor_courses_using_scope(self):
-        CourseInstructorRole(self.course_key).add_users(self.user)
-        claims = self.get_with_scope('course_instructor')
+    def test_request_global_staff_courses_using_scope(self):
```

Comment:
Can you add negative assertions in all the appropriate tests below?
For example, a user with `CourseInstructorRole` should not have `staff_courses` claim.
Or, is that covered by other tests?